### DISCUSSION TITLE
Keyboard shortcut for "clear cell output" in Notebook v7.0.3

### DISCUSSION THREAD

**Discussion by santhoshsram** [jupyter/notebook#7028] [Q&A]
I upgraded from notebook 6.5.4 to 7.0.3. In 7.0.3, how does one set a keyboard shortcut for clearing cell output. This feature was available in 6.5.4 under Help→Edit Keyboard Shortcuts. Although I am able to get to the edit keyboard shortcuts settings page (screenshot below), the "clear cell output" action is not available. How does one set a keyboard shortcut for
1. clearing current cell output
2. clearing all cell output

Was this feature yanked?

![Screenshot 2023-09-03 at 11 26 53 PM](https://github.com/jupyter/notebook/assets/5712605/5ef4c469-d3e2-411d-b7b3-3e230129a1b6)

**Comment by Eyobs-droid** [comment]
Yes, it is not directly available in 7.0.3, instead you can use the existing keyboard shortcut or the command pallete
